example/src/screens/CardScreen/CardScreen.js

```diff
@@ -149,8 +223,10 @@ class CardScreen extends React.Component {
           headerSize={this.setHeaderSize}
           headerHeight={sizes.cardScreenHeaderHeight}
           background={this.renderBackground(user)}
-        >
-          {this.renderContent(user)}
+          onEndReached={this.stickyHeaderEndReached}
+          onTopReached={this.stickyHeaderTopReached}>
+          {/* {this.renderContent(user)} */}
```

Comment:
This comment is necessary?

Reply:
Commented out code can be uncommented by users who don't know how to handle nested flatlist. Maybe it should also be documented in readme file?